src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php

```diff
@@ -138,7 +139,7 @@ public function process(ContainerBuilder $container)
                         $binding = $bindings[$bindingName];
 
                         list($bindingValue, $bindingId) = $binding->getValues();
-                        $binding->setValues([$bindingValue, $bindingId, true]);
+                        $binding->setValues([$bindingValue, $bindingId, true, BoundArgument::SERVICE_BIND, null]);
```

Comment:
why not keep the type and file returned the line just before?

Reply:
Done